composeApp/src/androidMain/kotlin/io/github/couchtracker/DbUtils.kt

```diff
@@ -0,0 +1,14 @@
+package io.github.couchtracker
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.State
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import app.cash.sqldelight.Query
+import app.cash.sqldelight.coroutines.asFlow
+import app.cash.sqldelight.coroutines.mapToList
+import kotlinx.coroutines.Dispatchers
+
+@Composable
+fun <T : Any> Query<T>.asListState(): State<List<T>> {
+    return this.asFlow().mapToList(Dispatchers.IO).collectAsStateWithLifecycle(emptyList())
```

Comment:
I wouldn't default to an empty list, but to something else that clearly indicates things aren't ready

Reply:
Yes I know, I want to create some sealed interface to do that next.